### DISCUSSION TITLE
set two Makers between two Interval and connect them with starigth line

### DISCUSSION THREAD

**Discussion by zeeshan-shabbir** [tradingview/lightweight-charts#1035] [Q&A]
hi again,
i am working on this awsome libary. i want to known if that(i:e title ) possible in this libary . i seacrh this featuer in issues and discussion but didnt find relevant information.
![Screenshot (2)](https://user-images.githubusercontent.com/80969033/155145080-2686d55c-d1c6-468d-bd86-f5a07495060a.png)

**Comment by manu-b-s** [comment]
As a workaround I came up with this, hope it helps:

```js
chart.addLineSeries({
      color: <color>,
      lineWidth: 1,
      lineStyle: LineStyle.Solid,
      lastValueVisible: false,
      priceLineVisible: false,
      crosshairMarkerVisible: false
    }).setData([
      {time: <time1>, value: <**sameValue**>},
      {time: <time2>, value: <**sameValue**>}
    ]);
```

If you need a second line above or below (as your screenshot), you will need to ADD this to the previous snippet:
```js
chart.addLineSeries({
      color: <color>,
      lineWidth: 1,
      lineStyle: LineStyle.Solid,
      lastValueVisible: false,
      priceLineVisible: false,
      crosshairMarkerVisible: false
    }).setData([
      {time: <time1>, value: <**anotherValue**>},
      {time: <time2>, value: <**anotherValue**>}
    ]);
```

Note that you can NOT just do the following:

```js
chart.addLineSeries({
      color: <color>,
      lineWidth: 1,
      lineStyle: LineStyle.Solid,
      lastValueVisible: false,
      priceLineVisible: false,
      crosshairMarkerVisible: false
    }).setData([
      {time: <time1>, value: <**sameValue**>},
      {time: <time2>, value: <**sameValue**>}
      {time: <time1>, value: <**anotherValue**>},   // ERROR: time1 already exists in the dataset
      {time: <time2>, value: <**anotherValue**>}    // ERROR: time2 already exists in the dataset
    ]);
```

**Reply by zeeshan-shabbir** [reply]
> As a workaround I came up with this, hope it helps:
> 
> ```js
> chart.addLineSeries({
>       color: <color>,
>       lineWidth: 1,
>       lineStyle: LineStyle.Solid,
>       lastValueVisible: false,
>       priceLineVisible: false,
>       crosshairMarkerVisible: false
>     }).setData([
>       {time: <time1>, value: <**sameValue**>},
>       {time: <time2>, value: <**sameValue**>}
>     ]);
> ```
> 
> If you need a second line above or below (as your screenshot), you will need to ADD this to the previous snippet:
> 
> ```js
> chart.addLineSeries({
>       color: <color>,
>       lineWidth: 1,
>       lineStyle: LineStyle.Solid,
>       lastValueVisible: false,
>       priceLineVisible: false,
>       crosshairMarkerVisible: false
>     }).setData([
>       {time: <time1>, value: <**anotherValue**>},
>       {time: <time2>, value: <**anotherValue**>}
>     ]);
> ```
> 
> Note that you can NOT just do the following:
> 
> ```js
> chart.addLineSeries({
>       color: <color>,
>       lineWidth: 1,
>       lineStyle: LineStyle.Solid,
>       lastValueVisible: false,
>       priceLineVisible: false,
>       crosshairMarkerVisible: false
>     }).setData([
>       {time: <time1>, value: <**sameValue**>},
>       {time: <time2>, value: <**sameValue**>}
>       {time: <time1>, value: <**anotherValue**>},   // ERROR: time1 already exists in the dataset
>       {time: <time2>, value: <**anotherValue**>}    // ERROR: time2 already exists in the dataset
>     ]);
> ```

what about marker ? marker is not set according to this lineseries .can i set marker according to value ?
